app/templates/components/ui-table/cell/admin/users/cell-first-name.hbs

```diff
@@ -1,21 +1,21 @@
-<span>{{extraRecords.firstName}}</span>
+<span>{{record}}</span>
 <div class="ui hidden divider"></div>
 <div class="ui horizontal compact basic buttons">
-  {{#ui-popup tagName='a' click=(action props.actions.moveToUserDetails record) content=(t 'View') class='ui icon button'}}
+  {{#ui-popup tagName='a' click=(action props.actions.moveToUserDetails extraRecords.id) content=(t 'View') class='ui icon button'}}
     <i class="unhide icon"></i>
   {{/ui-popup}}
-  {{#unless record.isSuperAdmin}}
-    {{#if (not record.deletedAt)}}
-      {{#ui-popup content=(t 'Edit') click=(action props.actions.openEditModal record) class='ui icon button'}}
+  {{#unless extraRecords.id.isSuperAdmin}}
```

Comment:
How will this work exactly? I don't understand how `id.isSuperAdmin` works? Am I missing something here?

Reply:
Nope, I need to watch `isSuperAdmin` too